ffc-ahwr-event/index.js

```diff
@@ -13,4 +14,8 @@ module.exports = async function (context, message) {
   } else {
     await saveMonitoring(context, event)
   }
+
```

Comment:
This need to be within the else after line 14. If not you will be sending all events to protective monitoring. The protective monitoring events are agreed with the SOC team and the events that are used on line 14 `saveMonitoring` are setup for those events.